src/model/RefreshSettings.ts

```diff
@@ -0,0 +1,7 @@
+class RefreshSettings {
+  autoRefresh: boolean = true;
+  interval: number = 30 * 1000; // hardcoded value for now
```

Comment:
Is there any reason why we don't just hardcode to a much smaller interval? I think a few seconds is all we should really wait for at this point.

[Yes, it will put more strain on the backend and p8s, but I think 30s is too long for now in our testing purposes].

Reply:
It was a coin toss.  How about 5 seconds?